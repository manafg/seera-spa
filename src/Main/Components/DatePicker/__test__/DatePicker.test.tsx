import { render } from "@testing-library/react";
import React from "react";
import ReactDOM from "react-dom";

import DatePickerComp from "../DatePicker";

import "@testing-library/jest-dom/extend-expect";
import rrender from "react-test-renderer";

describe("Snapshot tests", () => {
  const props = {
    onChange: jest.fn(),
    selectedDatePicker: "25/10/2019",
  };

  test("Matches existing Snapshot", () => {
    // @ts-ignore
    const tree = rrender.create(<DatePickerComp {...props}/> ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
