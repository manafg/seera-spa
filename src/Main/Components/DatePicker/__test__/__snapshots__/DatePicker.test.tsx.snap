// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests Matches existing Snapshot 1`] = `
<div
  className="react-datepicker-wrapper picker"
>
  <div
    className="react-datepicker__input-container"
  >
    <input
      className=""
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      readOnly={false}
      type="text"
      value="10/04/2021"
    />
  </div>
</div>
`;
